server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -50,6 +68,11 @@ case class NettyConfig(
 
   def socketBacklog(s: Int): NettyConfig = copy(socketBacklog = s)
 
+  def withRequestTimeout(r: Int): NettyConfig = copy(requestTimeout = Some(r.seconds))
```

Comment:
**suggestion:** These methods should also take just a `FiniteDuration`, let's hide underlying raw types as much as possible.